README.md

```diff
@@ -1,51 +1,182 @@
-# quickstart-sidecar-aws-ecs
+# Sidecar - Terraform AWS ECS
 
-## Deploy a single container sidecar on AWS ECS
+A quick start to deploy a sidecar to AWS EC2 using Terraform!
 
-This guide explains how to deploy a basic sidecar on AWS using
-the ECS service. This example provides only the necessary configuration to
-deploy a sidecar container. For a more complete example, please see the
+This guide explains how to deploy a basic sidecar on AWS using ECS Fargate.
```

Comment:
technically, they can provide their own cluster if they want, so there is nothing that restricts it to fargate

Reply:
Fixed in [9a4ba96](https://github.com/cyral-quickstart/quickstart-sidecar-aws-ecs/pull/13/commits/9a4ba9647766fe557121af74f52b7793eb812f92).